_api-reference/snapshots/create-repository.md

```diff
@@ -43,6 +43,14 @@ The following table lists parameters that can be used with both the `fs` and `s3
 Request field | Description
 :--- | :---
 `prefix_mode_verification` | When enabled, adds a hashed value of a random seed to the prefix for repository verification. For remote-store-enabled clusters, you can add the `setting.prefix_mode_verification` setting to the node attributes for the supplied repository. This field works with both new and existing repositories. Optional.
+`shard_path_type` | This setting is used to control the path stucture of shard level blobs. Defaults to fixed. There are 2 other possible values - 1. `hashed_prefix` 2. `hashed_infix`. More details are shared below. Optional.
+
+##### shard_path_type values
+1. FIXED - This keep the path structure in the existing hierarchical manner. eg - `<ROOT>/<BASE_PATH>/indices/<index-id>/0/<SHARD_BLOBS>`
```

Comment:
@ashking94: When adding these values to the API request, should they be lowercased or capitalized?